internal/route/node_filter.go

```diff
@@ -6,6 +6,10 @@ import (
 	"github.com/satsuma-data/node-gateway/internal/metadata"
 )
 
+const (
+	MaxAllowedBlocksBehindHead = 10
```

Comment:
If we want to guarantee clients will have access to data no staler than some amount of time, we probably want this configurable because of how different the block production times are. The plan is to run one gateway per chain right now, so having this configurable globally would be ok.

Reply:
Done - added a new `Routing` session of the config so that I can pass that into the filter factory and have it be responsible for that part of the config (e.g. handling defaults).